src/20_Components/amp-dynamic-css-classes.html

```diff
@@ -32,17 +35,15 @@
       display: none;
     }
     p {
```

Comment:
can you scope this to only apply to this sample

Reply:
Suggestion: use a `div` instead and add a class, e.g. `dynamic-css`

Reply:
done